crates/iota-protocol-config/src/lib.rs

```diff
@@ -37,7 +37,9 @@ pub const MAX_PROTOCOL_VERSION: u64 = 6;
 //            Add new gas model version to update charging of functions.
 //            Enable proper conversion of certain type argument errors in the
 //            execution layer.
-// Version 6: Variants as type nodes.
+// Version 6: Variants as type nodes. Enable smart ancestor selection for
+//            testnet. Enable probing for accepted rounds in round prober for
```

Comment:
Probably needs to be formatted, but this should be in a new line
```suggestion
//            testnet. 
//            Enable probing for accepted rounds in round prober for
```

Reply:
fixed